art/api/views.py

```diff
@@ -1,3 +1,27 @@
-from django.shortcuts import render
+from rest_framework.generics import ListCreateAPIView, RetrieveUpdateDestroyAPIView
+from rest_framework.response import Response
+from rest_framework import status
+from .models import Asset, Checkin
+from art.serializers import AssetSerializer, ChekinSerializer
```

Comment:
Please import the CheckinLogs and CheckoutLogs models and the Serializers as well.

Reply:
+1